libnetwork/drivers/bridge/bridge_linux.go

```diff
@@ -65,25 +66,27 @@ type configuration struct {
 	EnableUserlandProxy      bool
 	UserlandProxyPath        string
 	Rootless                 bool
+	AllowDirectRouting       bool
 }
 
 // networkConfiguration for network specific configuration
 type networkConfiguration struct {
-	ID                   string
-	BridgeName           string
-	EnableIPv4           bool
-	EnableIPv6           bool
-	EnableIPMasquerade   bool
-	GwModeIPv4           gwMode
-	GwModeIPv6           gwMode
-	EnableICC            bool
-	InhibitIPv4          bool
-	Mtu                  int
-	DefaultBindingIP     net.IP
-	DefaultBridge        bool
-	HostIPv4             net.IP
-	HostIPv6             net.IP
-	ContainerIfacePrefix string
+	ID                    string
+	BridgeName            string
+	EnableIPv4            bool
+	EnableIPv6            bool
+	EnableIPMasquerade    bool
+	GwModeIPv4            gwMode
+	GwModeIPv6            gwMode
+	EnableICC             bool
+	TrustedHostInterfaces []string
```

Comment:
We should write down the preconditions that other code depends upon so the next person knows not to break the contract.
```suggestion
	TrustedHostInterfaces []string // Interface names must not contain ':' characters
```

Reply:
Ah yes, just in case Linux starts allowing ':' chars!

Reply:
I'm referring to the bridge_store code which would not faithfully round-trip names that contain colon characters.